client/src/components/Header/Header.js

```diff
@@ -1,16 +1,26 @@
-import { Grid, makeStyles } from '@material-ui/core'
 import React from 'react';
-import Button from '@material-ui/core/Button';
+import { useLocation } from 'react-router-dom'
+import { Grid, makeStyles, Button } from '@material-ui/core'
+import headerBackgroundColor from './HeaderBgColor'
 
-const useStyles = makeStyles(theme => ({
-  root:{
-    position: 'absolute',
-    paddingLeft: theme.spacing(5),
-    paddingRight: theme.spacing(5),
-  },
-}));
 
 export default function Home() {
```

Comment:
So this was a copy pasta mistake from me. I forgot to change the name of this method. Could you change this to Header() and repush.